keyboards/kinesis/kint2pp/config.h

```diff
@@ -0,0 +1,72 @@
+#ifndef KINT2PP_CONFIG_H
+#define KINT2PP_CONFIG_H
+
+#include "../config.h"
+
+/* USB Device descriptor parameter */
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0002
+
+/* key matrix size */
+#define MATRIX_ROWS 15
+#define MATRIX_COLS  7 
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+*/
+#define MATRIX_ROW_PINS \
+  {			\
+   D7, /* ROW_EQL */	\
+   E0, /* ROW_1 */	\
+   E1, /* ROW_2 */	\
+   C0, /* ROW_3 */	\
+   C6, /* ROW_4 */	\
+   F6, /* ROW_5 */	\
+   D4, /* ROW_6 */	\
+   D2, /* ROW_7 */	\
+   D3, /* ROW_8 */	\
+   D0, /* ROW_9 */	\
+   B7, /* ROW_0 */	\
+   D1, /* ROW_MIN */	\
+   E6, /* ROW_ESC */	\
+   B4, /* ROW_F1 */	\
+   B2  /* ROW_F2 */	\
+  }
+
+#define MATRIX_COL_PINS \
+  {			\
+   E7, /* COL_0 */	\
+   F0, /* COL_1 */	\
+   F7, /* COL_2 */	\
+   B1, /* COL_3 */	\
+   B3, /* COL_4 */	\
+   B0, /* COL_5 */	\
+   D5 /* COL_6 */	\
+  }
```

Comment:
```suggestion
#define MATRIX_ROW_PINS { D7, E0, E1, C0, C6, F6, D4, D2, D3, D0, B7, D1, E6, B4, B2 }
#define MATRIX_COL_PINS { E7, F0, F7, B1, B3, B0, D5 }
```

Reply:
Can you explain why you made the suggestion to remove the signal names next to the pins?

I thought including the signal names here would be helpful to people working with this hardware.

I certainly can make the change, it just seems like a step backwards to me, hence my question :)

Thanks!

Reply:
It's kinda pointless as you can easily figure out which column/row is which just by looking at the layout macros.

Reply:
To me, it wasn’t obvious at all.

Anyway, done.